test/simple.js

```diff
@@ -4,31 +4,31 @@ var request = require('supertest');
 var should = require('should');
 var route = require('koa-route');
 var range = require('../');
-var koa = require('koa');
-var app = koa();
+var Koa = require('koa');
+var app = new Koa();
```

Comment:
What do you mean? Leaving the old version? In Koa 2, the app has to be initialized with the `new` keyword.